Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  desc "Visualization tool for elasticsearch"
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana/archive/v4.0.2.tar.gz"
+  sha256 "facb5b2eb31de38ebf6e30bb656c62a512e540623df98b33655f5b0da896dd27"
+  head "https://github.com/elastic/kibana.git"
+
+  depends_on "node"
+
+  def install
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
+    system "npm", "install"
+    system "npm", "install", "grunt-cli"
+    system "npm", "install", "bower"
+    system "./node_modules/.bin/bower", "install"
+    system "./node_modules/.bin/grunt", "build", "--force"
+
+    dist_dir = Pathname("build/dist/kibana")
```

Comment:
You don't need to lean on `Pathname` here - Should be just `dist_dir = buildpath/"build/dist/kibana"`.